haret/src/vr/states/common/normal.rs

```diff
@@ -67,12 +67,11 @@ pub fn handle_start_view<T: State>(state: T,
 }
 
 pub fn handle_get_state<T: State>(state: T,
-                                  msg: GetState,
+                                  GetState { epoch, view, op }: GetState,
```

Comment:
metajack talked to me about rust in 2012 and said, "it's the semantics of Erlang with the speed of C" and I was like "sold!" soooo, maybe pattern matching has been lurking around?